### PR TITLE
Preprod: movie: Evolution (2001)

### PR DESCRIPTION
**Pull request by paul-sheridan**
Reminiscent of the first Ghostbusters film, but they couldn't quite pull it off. Rating 2/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 1]
```diff
@@ -917,6 +917,55 @@ what if I was trapped inside a game [Ash discovered that the only way to exit th
...
+:: Major Themes
+the scientist profession [The story follows the local college science professors Ira Kane and Harry Block as they study newly discovered, fast-evolving extraterrestrial microbes that had hitched a ride to Earth on a meteor.]
+super accelerated Darwinian evolution [The extraterrestrial microbes evolved into complex dinosaur-like creatures in just a handful of days.]
+romantic love [The story built up to Ira and Allison falling in love in the end.]
+
+:: Minor Themes
+what it is like in university [The viewer is shown various stereotypical community college happenings: An apathetic Professor Ira Kane gave A's to nearly everyone in his biology class on their biology research projects, there was a wild college party, etc.]
+genetics [Ira discovered that the extraterrestrial life was nitrogen-based and had DNA that consisted of ten base pairs.]
+geology [Professor Harry Block made a point that he was a representative of the U.S. Geological Survey when he went to sample a recently crashed meteor.]
+brother and brother [The brothers Deke and Danny Donald were two of the dumbest students in Dr. Ira Kane's biology class.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:955]: Let me just point out that the comment doesn't actually state that we saw anything about the two brothers' relationship with each other.

File: notes/stories/film/film-scifi-2000s.st.txt [hunk 2]
```diff
@@ -917,6 +917,55 @@ what if I was trapped inside a game [Ash discovered that the only way to exit th
...
+biology [Ira discovered that the extraterrestrial life forms reproduced asexually via mitosis.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-2000s.st.txt:956]: so was it "sexually distinguished being"?

**Review by odinlake — APPROVED** [review]
couple of comments inline but nothing major

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt]: Indeed. I suppose we can do a bit better than that. New and improved motivation added.

File: notes/stories/film/film-scifi-2000s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:956]: I see we haven't quite defined the "being" in [speculative being](https://themeontology.org/theme.php?name=speculative%20being). Do we want things like asexually reproducing dinosaurs to be beings?

Comment (odinlake) [reply notes/stories/film/film-scifi-2000s.st.txt:956]: I think you've been using speculative being broadly, which is fine. I don't know about the asexually reproducing beings - i doubt we want to use it for bacteria, amorphous blobs etc. Maybe it is only interesting for things that resemble earth life and therefore makes us think "hmm, what would earth/society be like if things/we reproduced without sex?" If that is the case we could change it into a speculative society kind of of theme. Much like single gender society, come to think of it.

Comment (paul-sheridan) [reply notes/stories/film/film-scifi-2000s.st.txt:956]: I checked and all the usages of **sexually distinguished being** and its descendants pertain to societies of beings, which it seems to me in the interesting aspect here. I'm inclined to let the **biology** theme cover the asexual reproduction aspect of this story for the now. Then, down the road, review the **sexually distinguished being** usages with a view to identifying which correspond to novel societies that we might not have covered with other themes.

**Comment by odinlake** [comment]
Okay

**Pull request merged**